src/sailor/admin.lua

```diff
@@ -0,0 +1,333 @@
+	table.insert(t,name)
+	local password = "admin_password = " .."\""..page.POST.admin_password.."\""..",\n"
+	table.insert(t, password)
+	local theme ="theme = " .."\""..page.POST.theme.."\""..",\n"
+	table.insert(t,theme)
+	local layout = "layout = " .."\""..page.POST.layout.."\""..",\n"
+	table.insert(t,layout)
+	local parameter = "route_parameter = " .."\""..page.POST.route_parameter.."\""..",\n"
+	table.insert(t, parameter)
+	local error = "default_error404 = " .."\""..page.POST.default_error404.."\""..",\n"
+	table.insert(t, error)
+	local environment = "environment = " .."\""..page.POST.environment.."\""..",\n"
+	table.insert(t, environment)
+	local controller = "default_controller = " .."\""..page.POST.default_controller.."\""..",\n"
+	table.insert(t, controller)
+	local action = "default_action = " .."\""..page.POST.default_action.."\""..",\n"
+	table.insert(t, action)
+	local static = "default_static="..page.POST.default_static..",\n"
+	table.insert(t, static)
+	local autogen = "enable_autogen="..page.POST.enable_autogen..",\n"
+	table.insert(t, autogen)
+	local urls = "friendly_urls="..page.POST.friendly_urls..",\n"
+	table.insert(t, urls)
+	local trace = "hide_stack_trace="..page.POST.hide_stack_trace..",\n"
+	table.insert(t, trace)
+	local adminc = "enable_admin="..page.POST.enable_admin.."},\n"
+	table.insert(t, adminc)
+
+
+	local string2 = [=[ 
+	db = {
+		development = { ]=]
+
+	table.insert(t,string2)
+
+	local driver = "driver = " .."\""..page.POST.driver.."\""..",\n"
+	table.insert(t, driver)
+	local host = "host = " .."\""..page.POST.host.."\""..",\n"
+	table.insert(t, host)
+	local user = "user = " .."\""..page.POST.user.."\""..",\n"
+	table.insert(t, user)
+	local pass = "pass = " .."\""..page.POST.pass.."\""..",\n"
+	table.insert(t, pass)
+	local dbname = "dbname = " .."\""..page.POST.dbname.."\"".."}},\n"
+	table.insert(t, dbname)
+
+	local string3 = [==[
+		smtp = {
+	]==]
+
+	table.insert(t,string3)
+	local smtpserver = "server = " .."\""..page.POST.smtpserver.."\""..",\n"
+	table.insert(t, smtpserver)
+	local smtpuser = "user = " .."\""..page.POST.smtpuser.."\""..",\n"
+	table.insert(t, smtpuser)
+	local smtppass = "pass = " .."\""..page.POST.smtppass.."\""..",\n"
+	table.insert(t, smtppass)
+	local smtpfrom = "from = " .."\""..page.POST.smtpfrom.."\"".."},\n"
+	table.insert(t, smtpfrom)
+
+	local string4="lua_at_client = {"
+	table.insert(t, string4)
+
+	local vm = "vm = " .."\""..page.POST.lua_at_client.."\""..",},\n"
+	table.insert(t, vm)
+
+	local string5 = "debug = { \n"
+	table.insert(t, string5)
+
+	local debug = "inspect = "..page.POST.debug.."}}\n"
+	table.insert(t, debug)
+
+	local string6 = "return conf"
+	table.insert(t, string6)
+
+
+
+	local  file  = io.open(path, "w")
+	local writeconf = file:write(table.concat(t))
+	file:close()
+	if writeconf then
+		?>
+		<p>Configuration Edited Successfully</p>
+		<?lua
+	else
+		?>
+		<p>Error in Changing conf file</p>
+		<?lua
+	end
+
+
+
```

Comment:
Yes. I agree with you. Should I make an admin folder inside /src/sailor and then put views and logic separately ? I think we have to modify sailor.lua to work with the same.